CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 * [CHANGE] StoreGateway/Alertmanager: Add default 5s connection timeout on client. #6603
+* [CHANGE] Ingester: Make out-of-order native histogram ingestion automatically enabled when `-ingester.out-of-order-time-window > 0` and `-blocks-storage.tsdb.enable-native-histograms=true`.
```

Comment:
I don't think we need the changelog as your previous change has not been released.

Can you just modify the previous changelog entry of your PR #6626?